src/AnimatedEllipsis.js

```diff
@@ -66,6 +66,7 @@ export default class AnimatedEllipsis extends Component {
     Animated.timing(this._animation_state.dot_opacities[which_dot], {
       toValue: this._animation_state.target_opacity,
       duration: this.props.animationDelay,
+      useNativeDriver: false,
```

Comment:
Is there a reason you can't set it to `true`?

Reply:
I didn't test it with true.  I wanted to keep it in a separate PR